docs/revsets.md

```diff
@@ -53,6 +53,10 @@ Jujutsu attempts to resolve a symbol in the following order:
 3. Git ref
 4. Commit ID or change ID
 
+To override the priority, use the appropriate [revset function](#functions). For
+example, to resolve `xyz` as a change ID even if there happens to be a bookmark
+by the same name, use `change_id(xyz)`. This is particularly useful in scripts.
```

Comment:
(Non-blocking) If we really want to substitute for #6948, I'd prefer mentioning the `commit_id` function explicitly, so that people can figure out the purpose of `commit_id` by searching for it.

For example, we could change the first sentence:

> To override the priority, use a [revset function] such as `tag`, `change_id`, `commit_id`, etc. For example, ...

Another obvious option would be to keep the first sentence but use a commit id as an example. I'm OK with that, but I'm guessing you decided against that.

If neither of these options look OK to both you and Yuya, it's OK to not do this.

Reply:
Oops, fixed